lighthouse-cli/printer.ts

```diff
@@ -129,7 +153,7 @@ function createOutput(results: Results, outputMode: OutputMode): string {
       item.subItems.forEach(subitem => {
         // Get audit object from inside of results.audits under name subitem.
         // Coming soon events are not located inside of results.audits.
-        subitem = results.audits[subitem] || subitem;
+        subitem = results.audits[subitem as any] || subitem;
```

Comment:
It was added here: https://github.com/GoogleChrome/lighthouse/commit/703ded6b552d17ea070d8fe786f73c6bccecb106#diff-75200aeb03674022bdf57bef95101ce0

Looks like it was for comingSoon audits only, but yes as you identified, we don't want those. Looks like it can get nuked. (Should keep a comment to explain though)